package.json

```diff
@@ -57,14 +65,19 @@
     "@tsconfig/recommended": "^1.0.3",
     "@typescript-eslint/eslint-plugin": "^6.12.0",
     "@typescript-eslint/parser": "^6.12.0",
+    "@vitest/coverage-v8": "^2.0.4",
+    "@vitest/ui": "^2.0.4",
     "eslint": "^8.54.0",
     "eslint-config-prettier": "^9.0.0",
     "husky": "^9.0.6",
     "isomorphic-fetch": "^3.0.0",
-    "prettier": "^3.3.2",
+    "kolorist": "^1.8.0",
+    "prettier": "^3.1.0",
```

Comment:
Why reverting to old Prettier version?

Reply:
Probably incorrectly merged diff, will correct it thanks!